downloads.html

```diff
@@ -10,11 +10,9 @@
   - /other_distros/
 ---
 
-
-<h2>Thank you for trying out Minetest</h2>
+<h2>New user?</h2>
```

Comment:
As a heading this really lacks context. I'd much prefer to see something like "Need help getting started?"

Reply:
I'm open to a better title certainly.

Reply:
`Getting started` sounds good, just like the MT wiki article name.

Reply:
'Getting started' seems ok to me, will do.